Fix MRProgressView compiling issue

crazytonyli opened the pull request:
The library hasn't published an update in 7 years, I doubt there will be an update now to fix this compiling issue.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist: N/A

crazytonyli committed:
Use IO instead of File functions

mokagio commented on Podfile:
```diff
@@ -428,4 +428,16 @@ post_install do |installer|
+  if lines[810] == "        [((id)self.modeView) setProgress:self.progress];\n"
+    lines[810] = "        [((MRProgressView *)self.modeView) setProgress:self.progress];\n"
+    File.write(mrprogress_filepath, lines.join)
+  end
```
This is a nifty way of patching the issue 🤓 

Have you considered forking the repo? Might set us up for addressing future issues.

crazytonyli replied:
That didn't cross my mind. 🥲 

But I did think about removing this library, because it's only used in one single place: showing an activity indicator during uploading a featured image to a post's settings. I got lazy and decided to make this one line change instead. 😸

mokagio replied:
😄 

I came here to write the same thing. I found that same as you did (_sigh of relief_).

If we go in with the intention of removing this library, then I'd say it's okay to merge this approach. I wonder if we could get the help of some iOS UI dev 🤓

mokagio committed:
Fix MRProgressView compiling issue

The library hasn't published an update in 7 years, I doubt there will be
an update now to fix this compiling issue.

mokagio committed:
Update Podfile.lock
